src/node/NodeOptions.hs

```diff
@@ -222,6 +207,25 @@ addrNodeOption =
         metavar "HOST:PORT" <>
         help "Identifier of a node in a Kademlia network"
 
+tlsParamsOption :: Parser TlsParams
```

Comment:
@akegalj I also have little understanding of what's going on there, but after looking at this module I got quite sure what lwallet doesn't provide any endpoints:
https://github.com/input-output-hk/cardano-sl/blob/master/lwallet/Pos/Wallet/Light/Web/Server.hs